Split README.md file for Web and Web.AspNetCore

bakgerman opened the pull request:
Preview:

- https://github.com/bakgerman/NLog.Web/blob/nuget-enhancements/src/NLog.Web.AspNetCore/README.md
- https://github.com/bakgerman/NLog.Web/blob/nuget-enhancements/src/NLog.Web/README.md

snakefoot commented:
The snupkg-file is already enabled in the build.ps1

I think the ReadMe.md should be very basic (without build-status-links etc.), and should look very similar to this:

https://www.nuget.org/packages/NLog.Web.AspNetCore#readme-body-tab

Maybe with some links like these:
- https://github.com/NLog/NLog.Web/releases
- https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-6
- https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore
- https://github.com/NLog/NLog/wiki/Logging-troubleshooting

bakgerman commented:
Rolled back the snupkg changes to the csproj files.  I will work on the README.MD files as you have suggested.

bakgerman commented:
Updated the README.MD files as requested

bakgerman commented:
Is there any other changes that you would like me to perform?

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,15 @@
+- LoggerProvider for ASP.NET platform. 
+- Adds helpers and layout renderers for websites and web applications.
+
+Supported platforms:
```
All .NET Framework versions are supported as you can see here: https://www.nuget.org/packages/NLog.Web#supportedframeworks-body-tab

bakgerman replied:
changed

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,18 @@
+- For ASP.NET Core 2, .NET Standard 2.0 and .NET 4.6.1+
+
+Useful Links
+- [https://github.com/NLog/NLog.Web/releases](https://github.com/NLog/NLog.Web/releases)
```
Not sure how it works, but would it not be nicer with pretty-url-names ? Think it is easier to understand `Full changelog` than the raw url `https://github.com/NLog/NLog.Web/releases`

bakgerman replied:
I agree, I will adjust those accordingly.  I think the [] part if the pretty name, and the () part is the link.  I have done it before, should not take long.

bakgerman replied:
changed

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -11,8 +11,11 @@ Supported platforms:
-- [https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore)
-- [https://github.com/NLog/NLog/wiki/Logging-troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+- [Change Log](https://github.com/NLog/NLog.Web/releases)
+- [Tutorial](https://github.com/NLog/NLog/wiki/Tutorial)
```
Think this tutorial is mostly for console-applications, and should not be used as starting point for using NLog with AspNetCore-application.

bakgerman replied:
Completed

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -11,8 +11,11 @@ Supported platforms:
+- [Getting started with ASP.NET Core 6](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-6)
+- [Getting started with ASP.NET Core 5](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-5)
+- [Getting started with ASP.NET Core 3](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-3)
+- [Getting started with .NET Core 2 Console App](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-2---Console-application)
```
Think .NET Core 2 is now obsolete, and link should not be included.

bakgerman replied:
Removed

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -5,11 +5,18 @@
 
 Supported platforms:
 
+The .NET 3.5 build supports the following .NET releases
```
Instead of doing this splitting, then it should just state that `- .NET Framework 3.5 - 4.8` are supported.

bakgerman replied:
Changed

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,14 @@
+Supported platforms:
+
+.NET 3-5-4.8
+
```
With NLog v5 then it is very important to manually "register" NLog.Web in NLog.config, think this should be included:
```xml
  <!-- enable ASP.NET layout renderers -->
  <extensions>
    <add assembly="NLog.Web"/>
  </extensions>
```

bakgerman replied:
Complete for both README.md files

snakefoot commented:
I have no idea what the goal should be for the ReadMe in the nuget-package.

Would it make sense to expand a little on how to get help, besides the link to the basic troubleshooting-guide? Ex. using StackOverflow.com for NLog-related questions, and if wanting to contribute with new ideas / features then one is wellcome to create pull-requests ?

bakgerman commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,14 @@
+
+Supported platforms:
+
+.NET 3-5-4.8
```
Complete

bakgerman commented:
Yes, let me ponder that and I will add some additional content regarding the topics mentioned above.

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -17,3 +17,11 @@ Useful Links
 - [ASP.NET Core Layout Renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore)
 - [Logging Troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+
+Registration of NLog.Web in the NLog.config File
```
Think the "Useful links" should always be at the bottom. Any advice should be move aboe.

bakgerman replied:
complete

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -5,10 +5,17 @@
 - [ASP.NET Layout Renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web)
 - [Logging Troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+
+Registration of NLog.Web in the NLog.config File
```
Think the "Useful links" should always be at the bottom. Any advice should be moved above.

bakgerman replied:
complete

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,25 @@
+- For ASP.NET Core 2, .NET Standard 2.0 and .NET 4.6.1+
+
+Registration of NLog.Web in the NLog.config File
+	<!-- enable ASP.NET layout renderers -->
```
Also notice that NLog.Web is for ASP.NET Classic, and NLog.Web.AspNetCore is for ASP.NET Core

Would it make sense to use code-markup as xml ?

bakgerman replied:
yes, done

snakefoot replied:
Thank you for the markup, but the xml-code still says `NLog.Web` and not `NLog.Web.AspNetCore`

bakgerman replied:
sorry about that, fixed

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,20 @@
+
+Registration of NLog.Web in the NLog.config File
+	<!-- enable ASP.NET layout renderers -->
+	<extensions>
```
Would it make sense to use code-markup as xml ?

bakgerman replied:
done

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
````diff
@@ -0,0 +1,29 @@
+```xml
+	<!-- enable ASP.NET layout renderers -->
+	<extensions>
+		<add assembly="NLog.Web"/>
````
The xml-code still says NLog.Web and not NLog.Web.AspNetCore

bakgerman replied:
typing too fast...fixed again

snakefoot replied:
And also change `<add assembly="NLog.Web"/>` to `<add assembly="NLog.Web.AspNetCore"/>

bakgerman replied:
fixed

bakgerman commented:
Fix the Nlog.config example for Core, added additional lines of content for layout renders, layouts, and targets.  Added home page, PR, and stackoverflow links.

snakefoot commented:
Like the direction with a small introduction of what features are provided, and then providing helpful-links to get started.

I guess the small introduction should be "optimized" to cater for ASP.NET-developers and what they are looking for. You have by your own hand made it possible to extract all sorts of exciting details from the HttpRequests, which can be included in the logging-output as bonus details by updating NLog Target Layout, and without having to touch existing logging-statements.

bakgerman commented:
i have added 2 statements, 1 about ability to log all property of http context and host environment with only config change, 1 about included http modules / middlewares.

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform via extension methods for ILoggingProvider, IWebHostBuilder, and IHostBuilder.
```
Think Microsoft calls it [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) and not LoggerProvider.

bakgerman replied:
Completed

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform via extension methods for ILoggingProvider, IWebHostBuilder, and IHostBuilder.
+- Log all properties of HttpContext and IHostEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore) just by changing the NLog.config file.
```
"Extends logging output with details from the active HttpContext via layout renderers, just by changing NLog.config file."

bakgerman replied:
Completed

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+# NLog.Web
+
+- Logger Provider for ASP.NET platform.
+- Log all properties of HttpContext and HostingEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web) just by changing the NLog.config file.
```
"Extends logging output with details from the active HttpContext via layout renderers, just by changing NLog.config file."

bakgerman replied:
Completed

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+- LoggerProvider for ASP.NET Core platform via extension methods for ILoggingProvider, IWebHostBuilder, and IHostBuilder.
+- Log all properties of HttpContext and IHostEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore) just by changing the NLog.config file.
+- Capable to log to many destinations via [targets](https://nlog-project.org/config/?tab=targets).
+- Supports common [layouts](https://nlog-project.org/config/?tab=layouts), such as CVS, JSON, W3C ELF, and XML.
```
"Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog Layouts"

bakgerman replied:
Completed

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+- Logger Provider for ASP.NET platform.
+- Log all properties of HttpContext and HostingEnvironment via [layout renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web) just by changing the NLog.config file.
+- Capable to log to many destinations via [targets](https://nlog-project.org/config/?tab=targets).
+- Supports common [layouts](https://nlog-project.org/config/?tab=layouts), such as CVS, JSON, W3C ELF, and XML.
```
"Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog Layouts"

bakgerman replied:
Completed

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+- [Tutorial](https://github.com/NLog/NLog/wiki/Tutorial)
+- [ASP.NET Layout Renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web)
+- [Logging Troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+- [Support](https://stackoverflow.com/questions/tagged/nlog)
```
"Support" -> "Have a question?"

bakgerman replied:
Completed

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+- [Getting started with ASP.NET Core 3](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-3)
+- [ASP.NET Core Layout Renderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore)
+- [Logging Troubleshooting](https://github.com/NLog/NLog/wiki/Logging-troubleshooting)
+- [Support](https://stackoverflow.com/questions/tagged/nlog)
```
"Support" -> "Have a question?"

bakgerman replied:
Completed

bakgerman commented:
New version with latest comments from above included

snakefoot commented:
For me it looks good. Do you have any comments @304NotModified ?

snakefoot commented:
@bakgerman Can these 2 files also be deleted:

- https://github.com/NLog/NLog.Web/blob/master/src/NLog.Web/readme.txt
- https://github.com/NLog/NLog.Web/blob/master/src/NLog.Web.AspNetCore/readme.txt

Since replaced by the new README.md ?

bakgerman commented:
Older readme.txt files removed

snakefoot commented:
Maybe we should add some image-badges from SonarQube? (Just below the first headline)
```
[![](https://sonarcloud.io/api/project_badges/measure?project=nlog.web&branch=master&metric=reliability_rating)](https://sonarcloud.io/dashboard/?id=nlog.web&branch=master) 
[![](https://sonarcloud.io/api/project_badges/measure?project=nlog.web&branch=master&metric=sqale_rating)](https://sonarcloud.io/dashboard/?id=nlog.web&branch=master) 
[![](https://sonarcloud.io/api/project_badges/measure?project=nlog.web&branch=master&metric=bugs)](https://sonarcloud.io/dashboard/?id=nlog.web&branch=master) 
[![](https://sonarcloud.io/api/project_badges/measure?project=nlog.web&branch=master&metric=vulnerabilities)](https://sonarcloud.io/dashboard/?id=nlog.web&branch=master) 
```

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) for ASP.NET Core platform via extension methods for IHostBuilder, and IWebHostBuilder.
```
Instead of having this as feature-bullet, then this should be the introduction (below the headline)

```
Integrates NLog as [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) for the ASP.NET Core platform by just calling `UseNLog()` on the host-builder.
```

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) for ASP.NET Core platform via extension methods for IHostBuilder, and IWebHostBuilder.
+- Extends logging output with details from the active HttpContext via NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), just by changing NLog.config file.
```
Think `Enrich` is better:
```
- Enrich logging output with additional details from active HttpContext using NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), by just updating the NLog configuration.
```

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+
+- [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) for ASP.NET Core platform via extension methods for IHostBuilder, and IWebHostBuilder.
+- Extends logging output with details from the active HttpContext via NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), just by changing NLog.config file.
+- Capable to log to many destinations via NLog [Targets](https://nlog-project.org/config/?tab=targets).
```
```
- Logging output can be routed to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)
```

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+- Extends logging output with details from the active HttpContext via NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), just by changing NLog.config file.
+- Capable to log to many destinations via NLog [Targets](https://nlog-project.org/config/?tab=targets).
+- Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).
+- Supports advanced HTTP pipeline interception via provided IMiddleware classes, including logging of HTTP request body and HTTP response body.
```
```
- Supports middleware injection for [HTTP Request Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Request-Logging)
```

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,35 @@
+- Capable to log to many destinations via NLog [Targets](https://nlog-project.org/config/?tab=targets).
+- Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).
+- Supports advanced HTTP pipeline interception via provided IMiddleware classes, including logging of HTTP request body and HTTP response body.
+- To add new features [pull requests](https://github.com/NLog/NLog.Web/pulls) are always welcome.
```
```
- Contributions are always wellcome, and [pull requests](https://github.com/NLog/NLog.Web/pulls) are appreciated.
```

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+
+Supported platforms:
+
+.NET 3.5 - 4.8
```
Can it become a bullet:
```
- .NET 3.5 - 4.8
```

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+# NLog.Web
+
+- Logger Provider for ASP.NET platform.
+- Extends logging output with details from the active HttpContext via NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web), just by changing NLog.config file.
```
Think `Enrich` is better:
```
- Enrich logging output with additional details from active HttpContext using NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), by just updating the NLog configuration.
```

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+
+- Logger Provider for ASP.NET platform.
+- Extends logging output with details from the active HttpContext via NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web), just by changing NLog.config file.
+- Capable to log to many destinations via NLog [Targets](https://nlog-project.org/config/?tab=targets).
```
```
- Logging output can be routed to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)
```

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+- Extends logging output with details from the active HttpContext via NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web), just by changing NLog.config file.
+- Capable to log to many destinations via NLog [Targets](https://nlog-project.org/config/?tab=targets).
+- Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).
+- Supports advanced HTTP pipeline interception via provided IHttpModule classes, including logging of HTTP request body.
```
```
- Supports HttpModule injection for [HTTP Request Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Request-Logging)
```
Think this bullet should be moved up the list. Just below the bullet with NLog LayoutRenderers

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+- Capable to log to many destinations via NLog [Targets](https://nlog-project.org/config/?tab=targets).
+- Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).
+- Supports advanced HTTP pipeline interception via provided IHttpModule classes, including logging of HTTP request body.
+- To add new features [pull requests](https://github.com/NLog/NLog.Web/pulls) are always welcome.
```
```
- Contributions are always wellcome, by just opening a [pull requests](https://github.com/NLog/NLog.Web/pulls).
```

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,30 @@
+# NLog.Web
+
+- Logger Provider for ASP.NET platform.
```
Think this bullet should become a short introduction (below the headline, without being a bullet)

snakefoot replied:
> Integrates NLog with the System.Web.HttpContext, and providing features like:

snakefoot commented:
Since README.md will replace current package-description:

- https://github.com/NLog/NLog.Web/blob/master/src/NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj

Then I think `<description>` should be updated to this:

> Integrates NLog as provider for Microsoft.Extensions.Logging, and provides NLog LayoutRenderers to enrich logging output with details from HttpContext.

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,43 @@
+
+- Enrich logging output with additional details from active HttpContext using NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), by just updating the NLog configuration.
+- Supports middleware injection for [HTTP Request Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Request-Logging) and [HTTP Response Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Response-Body-Capture).
+- Logging output can be routed to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)
```
` - Routing logging output to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)`

snakefoot commented on src/NLog.Web.AspNetCore/README.md:
```diff
@@ -0,0 +1,43 @@
+- Enrich logging output with additional details from active HttpContext using NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), by just updating the NLog configuration.
+- Supports middleware injection for [HTTP Request Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Request-Logging) and [HTTP Response Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Response-Body-Capture).
+- Logging output can be routed to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)
+- Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).
```
` - Rendering logging output into standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).`

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,38 @@
+- Enrich logging output with additional details from active HttpContext using NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), by just updating the NLog configuration.
+- Supports HttpModule injection for [HTTP Request Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Request-Logging)
+- Logging output can be routed to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)
+- Supports logging output in standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).
```
` - Rendering logging output into standard formats like JSON, CVS, W3C ELF and XML using NLog [Layouts](https://nlog-project.org/config/?tab=layouts).`

snakefoot commented on src/NLog.Web/README.md:
```diff
@@ -0,0 +1,38 @@
+
+- Enrich logging output with additional details from active HttpContext using NLog [LayoutRenderers](https://nlog-project.org/config/?tab=layout-renderers&search=package:nlog.web.aspnetcore), by just updating the NLog configuration.
+- Supports HttpModule injection for [HTTP Request Logging](https://github.com/NLog/NLog.Web/wiki/HTTP-Request-Logging)
+- Logging output can be routed to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)
```
` - Routing logging output to multiple destinations via the available NLog [Targets](https://nlog-project.org/config/?tab=targets)`

snakefoot commented:
These are my last nit-picks, and then this pull-request be merged right after.

bakgerman commented:
Thank you, for the comments.

snakefoot commented on README.md:
```diff
@@ -1,93 +0,0 @@
-![NLog](https://raw.githubusercontent.com/NLog/NLog.github.io/master/images/NLog-logo-only_small.png)
```
Oops didn't see that this pull-requests also removes the main-page for the entire NLog.Web-repo. Please update the pull-request to not delete the file:

- https://github.com/NLog/NLog.Web/blob/master/README.md

bakgerman replied:
Certainly

bakgerman replied:
Completed

snakefoot commented:
@bakgerman Thank you again for all the hard work.

Thinking about preparing the last NLog-release for this year. Do you have any last code-changes?

bakgerman commented:
No for this year, think health check pull request will be later in January.

snakefoot commented:
Nuget-packages are now live:
- https://www.nuget.org/packages/NLog.Web.AspNetCore#readme-body-tab
- https://www.nuget.org/packages/NLog.Web#readme-body-tab

Think they look good!
